libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -48,9 +48,9 @@
 <ng-template #empty>
   <div class="empty">
     <p>
-      Try searching for a topic, for example "<a (click)="searchExample()"
+      Try searching for a topic, for example "<a href="#" (click)="searchExample()"
```

Comment:
For proper accessibility, buttons should be represented with a `<button>` tag rather than an anchor tag with a `(click)` listener.

Reply:
The  anchor tag <a> is replaced with  <button> tag .